src/Extension/AccountReset/SecurityAdminExtension.php

```diff
@@ -135,12 +139,13 @@ protected function sendResetEmail($member)
                 ->addData('AccountResetLink', $this->getAccountResetLink($member, $token))
                 ->addData('Member', $member)
                 ->setTo($member->Email);
-
-            return $email->send();
+            $email->send();
         } catch (Exception $e) {
             $this->logger->info('WARNING: Account Reset Email failed to send: ' . $e->getMessage());
             return false;
         }
+
+        return true;
     }
```

Comment:
`Email::send()` now return `void`. We'll consider the Email sending a success if no exceptions are thrown.